### PR TITLE
Nicolas

### PR DESCRIPTION
**Pull request by calamitoso**
@bruno-morel Hey Bruno! Here it is. I'll leave some comments in the code, but please ask any questions you might have

Since I based my work on a Create React App setup, you'll see here a bunch of CRA files that I did not touch; here is a[ good diff ](https://github.com/calamitoso/amaze-us/compare/e41f29...bf981d?expand=1) focusing on the code I actually wrote.

I have deployed the app here https://calamitoso.github.io/amaze-us/

A good point to begin is the README here https://github.com/calamitoso/amaze-us/tree/nicolas/uiux/challenge-1

Enjoy!

### CODE REVIEW

File: uiux/challenge-1/src/api/musixmatch/provider.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      }
+    });
+
+    let promise = fetch(route);
+    promise = fetchHandler(promise);
+    promise = extractResponseBody(promise);
+    return promise;
+  }
+
+  // /**
```
Comment (calamitoso) [inline uiux/challenge-1/src/api/musixmatch/provider.js]: These are the remains of an idea for an alternate navigation path

File: uiux/challenge-1/src/api/musixmatch/adapters.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+export const artistAdapter = ({artist}) => ({
```
Comment (calamitoso) [inline uiux/challenge-1/src/api/musixmatch/adapters.js]: This is a good point to begin reviewing

File: uiux/challenge-1/src/components/presentational/artist-list/index.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                display: 'inline-block',
+                height: '4rem',
+                marginRight: '0.5rem',
+                textDecoration: 'none',
+                width: '4rem',
+              }}
+            ></span>
+            <span                
+              style={{
+                color: `rgba(255, 255, 255, ${1 - (index / 3)})`,
```
Comment (calamitoso) [inline uiux/challenge-1/src/components/presentational/artist-list/index.js]: With more time I would work on refactoring styles to facilitate reuse. I prioritized delivery

**Comment by bruno-morel** [comment]
Thank you Nicolas ! :) we'll contact you soon ;-)

**Pull request closed without merging**